### ISSUE TITLE
select last sensor box as default for new widget

### ISSUE THREAD

**Issue by silberzwiebel** [CodeforKarlsruhe/opensense#18]
so one does not have to zoom into the same area of the world every time one wants to add a new widget

**Comment by dbaelz** [comment]
Good idea. The constant zooming to find the desired location is really annoying. Another idea is to focus the current location in different zoom level. I'll look up what options the mapbox widget provides.
